xwayland/xwm.h

```diff
@@ -0,0 +1,84 @@
+#ifndef XWAYLAND_INTERNALS_H
+#define XWAYLAND_INTERNALS_H
+#include <wayland-server-core.h>
+#include <wlr/xwayland.h>
+
+/* wlc's atom list:
```

Comment:
Looked a bit and didn't find anything that looks better to me here. Obviously a difference in opinion, I'll leave as is as it is my personal preference and defer to @SirCmpwn if he cares (pick one of current enum + static string array all in our header, or weston-style long xwm struct defining all atoms we use in header + array with a substruct that holds string/positions in that struct defined within the code later)